content/content.go

```diff
@@ -121,6 +122,7 @@ func (s *RulesWithContentStorage) GetRuleIDs() []string {
 		ruleIDs = append(ruleIDs, ruleContent.Plugin.PythonModule)
 	}
 
+	sort.Strings(ruleIDs)
```

Comment:
I think it's better to sort when comparing in tests

Reply:
I think about it and you are right. I just modified it